translations/bnpchs-final-2018-chairoikoguma.html

```diff
@@ -0,0 +1,107 @@
+<h3>Description</h3>
+
+<p style="text-align: justify;">
+  Chairoikoguma (otherwise known as Koguma) is a bear that loves honey a lot. At one time in the honey forest, while wandering around peacefully, Koguma found the 
+  location of the tastiest honey in the world. Koguma wants to stay as long as possible while eating honey.
+</p>
+
+<p style="text-align: justify;">
+  Unfavorably, Jelix, the landlord of the honey forest, despises the honey snatching. The forest populates bees. As soon as Jelix notices Koguma 
+  approaching the honey, he sends the bees from the hives to strike Koguma. Koguma wants to eat honey as long as possible, but he hates bee sting. Therefore Koguma eats 
+  honey for maximum possible time, then goes home safely.
+</p>
+
+<p style="text-align: justify;">
+  The honey forest can be illustrated as $N \times N$ meters grid. With symbols as follow:
+  <ul>
+  <li>The symbol <code>K</code> represents the initial position of Koguma,</li>
+  <li>The symbol <code>.</code> represents an empty field, </li>
+  <li>The symbol <code>#</code> represents a woods that can be passed by neither of Koguma nor the bees,</li>
+  <li>The symbol <code>@</code> represents Koguma's house, </li>
+  <li>The symbol <code>H</code> represents a beehive. There can be more than one beehive in the forest.</li>
+  </ul>
+</p>
+
+<h4>Koguma Movements</h4>
+
+<p style="text-align: justify;">
+  In one second, Koguma can move <b>at most</b> $S$ steps in four directions: left, right, up and down. Each step could has different direction. For example if 
+  $S = 3$, then in 1 second Koguma can move with such direction: left-right-top, left-top, top, right-right-right, and so on.
```

Comment:
```suggestion
  $S = 3$, then in one second, Koguma can move with these directions: left-right-up, left-up, up, right-right-right, and so on.
```

biasakan angka ditulis penuh hurufnya

"top" itu menunjukkan posisi atas, bukan arah